### PR TITLE
docs(deploy-mayastor): changed existing info on helm chart

### PR DESCRIPTION
**Pull request by AJDatacore**

### CODE REVIEW

**Review by Abhinandan-Purkait — APPROVED** [review]

File: quickstart/deploy-mayastor.md [hunk 1]
```diff
@@ -11,9 +11,8 @@ The steps and commands which follow are intended only for use in conjunction wit
 ## Installation via helm
 
 {% hint style="info" %}
-Installing Mayastor via the Helm chart sets the StorageClass as 'default' for Loki and etcd StatefulSets. This will work in clusters which have a StorageClass named 'default'. If there is no 'default' StorageClass in the cluster, storage provisioning will fail for Loki and etcd. In such cases, one can change the StorageClass to 'manual' which will provision a PersistentVolume of type hostPath. To do this, make the following changes in the `values.yaml` file:
-- loki-stack.persistence.storageClassName: manual
-- etcd.persistence.storageClass: manual
+The Mayastor Helm chart now includes the Dynamic Local Persistent Volume (LocalPV) provisioner as the default option for provisioning storage to etcd and Loki. This simplifies storage setup by utilizing local volumes within your Kubernetes cluster.
+For etcd, the chart uses the `mayastor-etcd-localpv` storage class, and for Loki, it utilizes the `mayastor-loki-localpv` storage class. These storage classes are bundled with the Mayastor chart, ensuring that your etcd and Loki instances are configured to use openEbs localPV volumes efficiently.
```
Comment (tiagolobocastro) [inline quickstart/deploy-mayastor.md]: We should also add a warning that by default these PV's use `/var/local/` which must be preserved?
 @Abhinandan-Purkait

File: quickstart/deploy-mayastor.md [hunk 2]
```diff
@@ -11,9 +11,9 @@ The steps and commands which follow are intended only for use in conjunction wit
...
-Installing Mayastor via the Helm chart sets the StorageClass as 'default' for Loki and etcd StatefulSets. This will work in clusters which have a StorageClass named 'default'. If there is no 'default' StorageClass in the cluster, storage provisioning will fail for Loki and etcd. In such cases, one can change the StorageClass to 'manual' which will provision a PersistentVolume of type hostPath. To do this, make the following changes in the `values.yaml` file:
-- loki-stack.persistence.storageClassName: manual
-- etcd.persistence.storageClass: manual
+The Mayastor Helm chart now includes the Dynamic Local Persistent Volume (LocalPV) provisioner as the default option for provisioning storage to etcd and Loki. This simplifies storage setup by utilizing local volumes within your Kubernetes cluster.
+For etcd, the chart uses the `mayastor-etcd-localpv` storage class, and for Loki, it utilizes the `mayastor-loki-localpv` storage class. These storage classes are bundled with the Mayastor chart, ensuring that your etcd and Loki instances are configured to use openEbs localPV volumes efficiently. 
+`/var/local/{{ .Release.Name }}` paths should be persistent accross reboots.
```
Comment (tiagolobocastro) [inline quickstart/deploy-mayastor.md:16]: Isn't it `/var/local/localpv-hostpath/{{ .Release.Name}}/` ?

Comment (Abhinandan-Purkait) [reply quickstart/deploy-mayastor.md:16]: Yes

Comment (tiagolobocastro) [reply quickstart/deploy-mayastor.md:16]: @AJDatacore please amend the path, otherwise LGTM

Comment (tiagolobocastro) [reply quickstart/deploy-mayastor.md:16]: @AJDatacore @Abhinandan-Purkait has made some changes so now for develop it should be
`/var/local/{{ .Release.Name }}`
But please raise another PR for release/2.4 where it should be this path:
/var/local/localpv-hostpath/{{ .Release.Name}}/

Comment (AJDatacore) [reply quickstart/deploy-mayastor.md:16]: @tiagolobocastro Can you please approve this PR

**Review by tiagolobocastro — APPROVED** [review]

**Pull request merged**